internal/configs/ingress_test.go

```diff
@@ -126,6 +126,26 @@ func TestPathOrDefaultReturnActual(t *testing.T) {
 	}
 }
 
+func TestGenerateIngressPathExact(t *testing.T) {
```

Comment:
PathTypePrefix is not tested. It is an omission.

Also, Since we'll have 3 different input (and the forth for `nil` check as per previous suggestion), better to use a table driven test.  For example, take a look at `TestGeneratePath` from internal/configs/virtualserver_test.go:4481.